.github/workflows/tests.yml

```
@@ -87,7 +87,7 @@ jobs:
             omp_num_threads: 2
 
           - name: "Coverage"
-            os: ubuntu-18.04
+            os: ubuntu-latest
```

This build is failing because `latest` has PETSc/SLEPc 3.15 -- so:

```suggestion
            os: ubuntu-20.04
```